### PR TITLE
Dev to Main

### PR DESCRIPTION
**Pull request by brianpippin**

### CODE REVIEW

**Comment by kaitlyn-sharo** [comment]
Just a note that we'll need to merge in the ingest branch with this for everything to work

File: dashboard-ui/src/components/Survey/dynamic.jsx [hunk 1]
```diff
@@ -134,7 +134,10 @@ const Dynamic = ({ patients, situation, supplies, decision, dmName, actions, exp
                                     <Accordion.Body>
                                         <ListGroup>
                                             {actions.map((action, index) => (
-                                                <ListGroup.Item key={"action-" + index}>{action}</ListGroup.Item>
+                                                <ListGroup.Item key={"action-" + index} style={{
+                                                    "font-weight": action.includes('Update:') || action.includes('Note:') ? "700" : "500",
```
Comment (dereknop) [inline dashboard-ui/src/components/Survey/dynamic.jsx:138]: This bit here was causing console errors for me when taking the survey. 
Screenshot 2024-07-10 at 2 54 01 PM
Screenshot 2024-07-10 at 2 53 30 PM

Changing them to `fontWeight` and `backgroundColor` resolved for me locally.

**Review by dereknop — COMMENTED** [review]
Everything looked good to me besides the one minor console error I ran into. The survey looks good along with the upload to mongo (tracking info and answers appearing as expected). I also tested the text-based scenarios with the TA1 servers running locally and was able to get the alignment scores written to mongo as expected.

File: dashboard-ui/src/components/Survey/dynamic.jsx (hunk 1 above)
Comment (brianpippin) [reply dashboard-ui/src/components/Survey/dynamic.jsx:138]: Did you want to commit that change?

**Comment by brianpippin** [comment]
> Just a note that we'll need to merge in the ingest branch with this for everything to work

Roger Roger.

File: dashboard-ui/src/components/Survey/dynamic.jsx (hunk 1 above)
Comment (dereknop) [reply dashboard-ui/src/components/Survey/dynamic.jsx]: Just pushed it

**Review by dereknop — APPROVED** [review]

**Review by Ayal-Ciobotaru — APPROVED** [review]

**Pull request merged**